modules/microshift-firewall-config.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-firewall-config_{context}"]
+= Configuring the firewall
+
+Firewalls are not required in {product-title}, but using a firewall can prevent undesired access to the {product-title} API. Install and configure the `firewalld` service by using the following procedures.
+//Q: Are there networking prerequisites for this procedure, such as having already installed the OpenShift DNS Operator?
+[id="microshift-firewall-install_{context}"]
+== Installing the firewalld service
+.Procedure
+
+. To install the `firewalld` service:
++
+[source,terminal]
+----
+$ sudo dnf install -y firewalld
+----
+
+. To initiate the firewall:
++
+[source,terminal]
+----
+$ sudo systemctl enable firewalld --now
+----
+
+[IMPORTANT]
+====
+{product-title} pods must have access to the internal CoreDNS component and API servers.
+====
+
+[id="microshift-required-settings_{context}"]
+== Required settings
+An IP range for pods is a required part of the firewall configuration. You can use the default values or customize the IP range. You must also configure pod access to the internal CoreDNS component.
+
+.Required settings
+[cols="1,1",options="header"]
+|===
+^| IR Range ^| Description
+
+|10.42.0.0/16
+|Host network pod access to CoreDNS and {product-title} API
+
+|169.254.169.1
+|Host network pod access to {product-title} API Server
+|===
+
+.Procedure
+
+. Configure the IP range with either default or custom values:
+
+.. To use default values for the IP range:
++
+[source,terminal]
+----
+$ sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16
+----
+
+.. To use custom values for the IP range:
++
+[source,terminal]
+----
+$ sudo firewall-cmd --permanent --zone=trusted --add-source=<custom IP range>
+----
+
+. To set the CoreDNS server:
```

Comment:
```suggestion
. To set the CoreDNS server:
```

This isn't exactly to allow the coredns server. This is to allow pods talking to the k8s API and other services, that IP address 169.254.169.1 is used internally in OVN to route the packets from pods to host. I am not sure what would be the best way to explain this to customers.

I would probably say we need to enable this rule to allow traffic from the pods coming through the ovn gateway.